src/test/scala/temple/builder/DatabaseBuilderTest.scala

```diff
@@ -15,4 +23,88 @@ class DatabaseBuilderTest extends FlatSpec with Matchers {
     val createQuery = DatabaseBuilder.createServiceTables("temple_user", BuilderTestData.sampleComplexService)
     createQuery shouldBe DatabaseBuilderTestData.sampleComplexServiceCreate
   }
+
+  it should "correctly build endpoint create queries" in {
+    val queries = DatabaseBuilder.buildQuery(
+      "test-service",
+      BuilderTestData.sampleService,
+      Set(CRUD.Create),
+      IDAttribute("id", AttributeType.UUIDType),
+      CreatedByAttribute.None,
+    )
+    queries.keys should contain(CRUD.Create)
+    queries(CRUD.Create) should be(DatabaseBuilderTestData.sampleInsertStatement)
```

Comment:
```suggestion
    queries.keys should contain CRUD.Create
    queries(CRUD.Create) should be DatabaseBuilderTestData.sampleInsertStatement
```
It just seems too weird that `be` is a function